accounts/abi/argument.go

```diff
@@ -33,6 +33,7 @@ func (a *Argument) UnmarshalJSON(data []byte) error {
 	var extarg struct {
 		Name string
 		Type string
+		Indexed bool
```

Comment:
Why is Indexed being added to the argument list? Does being indexed in the events not suffice?